src/Main.gd

```diff
@@ -16,14 +16,19 @@
 
 extends Node2D
 
+const pack_download_div_scene = preload("res://pad_ui/pack_download_div.tscn")
```

Comment:
Is this used?

Reply:
Nice catch, this div is only instance by PackDownloadPage. Remove this line here.